package-lock.json

```diff
@@ -1,8 +1,4848 @@
 {
   "name": "harmony-marketplace-sdk",
   "version": "1.0.0",
-  "lockfileVersion": 1,
```

Comment:
same as the other PR, I don't think there should be a package lock diff

Reply:
Copy that!